src/xmlcli/XmlCliLib.py

```diff
@@ -792,12 +792,13 @@ def ConfXmlCli(SkipEnable=0):
       try:
         from .tools.restricted import EnableXmlCli as exc
       except (ModuleNotFoundError, ImportError) as e:
-        from .tools import EnableXmlCli as exc
-      except ImportError:
-        log.error(f'Import error on EnableXmlCli, current Python version {sys.version}')
-        CloseInterface()
-        LastErrorSig = 0x13E4  # import error
-        return 0xF
+        try:
```

Comment:
Interesting observation, because it appears so that this shall not be causing any issue as no changes reformed in python 3.12 w.r.t https://docs.python.org/3/library/exceptions.html#ModuleNotFoundError and https://docs.python.org/3/library/exceptions.html#ModuleNotFoundError